src/main/java/org/reactome/addlinks/EnsemblBioMartUtil.java

```diff
@@ -0,0 +1,146 @@
+package org.reactome.addlinks;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
```

Comment:
` javax.json...` is already in use in a number of other places. Why not use it?

Reply:
See response <a href="https://github.com/reactome/add-links/pull/134#discussion_r373660412">here</a> from similar question above